adoc/admin-software-installation.adoc

```diff
@@ -98,22 +98,24 @@ sudo zypper in ceph-common xfsprogs
 [[helm-tiller-install]]
 === Installing Helm
 
-As of {productname} {productversion}, Helm is part of the {productname} package repository, so to use this,
+As of {productname} {productversion}, Helm 2 is part of the {productname} package repository, so to use this,
 you only need to run the following command from the location where you normally run `skuba` commands:
 
 [source,bash]
 ----
 sudo zypper install helm
 ----
 
-Helm 2 is the default for {productname} 5. Helm 3 is offered as an alternate tool and may be installed in parallel to aid migration. Helm 3 may be selected by installing from zypper then selecting the alternative.
+Helm 2 is the default for {productname} 5. Helm 3 is offered as an alternate tool and may be installed in parallel to aid migration.
 
 [source,bash]
 ----
 sudo zypper install helm3
 sudo update-alternatives --set helm /usr/bin/helm3
 ----
 
+Unless you are migrating from {productname} 4 or have legacy Helm charts that only work with Helm 2, please use Helm 3.
```

Comment:
Suggest to word ot like

```Unless you are upgrading from {productname} 4 with Helm 2 charts already deployed... ```


Potentially this could be within the following warning box as well?

Reply:
Thanks for the suggestion. I pushed another commit to incorporate the wording.